portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,25 @@
+body {
+  background: #202020;
+  color: white;
+}
+
 #content {
   margin-left: auto;
   margin-right: auto;
   width: 650px;
 }
 
+#intro {
+  border: 1px solid black;
+}
+
+img {
+  width: 350px;
+  height: 400px;
+  border: 1px solid white;
+  box-shadow: 0 19px 30px rgba(0,0,0,0.30), 0 15px 12px rgba(0,0,0,0.22);
```

Comment:
nit - Per the [style guide](https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order):
`Put declarations in alphabetical order in order to achieve consistent code in a way that is easy to remember and maintain.`

Reply:
Noted (and updated)! Thanks for the feedback!

Reply:
Did the commit message update? Looks like it still has the typo. Maybe try [this](https://help.github.com/en/github/committing-changes-to-your-project/changing-a-commit-message).

![image](https://user-images.githubusercontent.com/2066940/85443347-01c13980-b546-11ea-8e1c-d26898ccd4f3.png)

Reply:
I'm not sure if I can change the previous commit's wording unfortunately